res/values-ar/strings.xml

```diff
@@ -1211,15 +1211,15 @@
     <string name="PERMANENT_BOOSTS">مسرع دائم</string>
     <string name="Video_failed_to_load_">فشل تحميل الفيديو!</string>
     <string name="Video_Ready_">الفيديو جاهز!</string>
-    <string name="GIVE_PLASMA">"مَنح بلازما"</string>
+    <string name="GIVE_PLASMA">"اهداء بلازما"</string>
     <string name="Specify_amount_">حدد المبلغ:</string>
     <string name="Specify_amount_again_to_confirm_">حدد المبلغ مرة اخرى للإكمال:</string>
     <string name="Amounts_did_not_match_">المبلغ غير مُتطابق!</string>
     <string name="RECEIVE_PLASMA">تلقي البلازما</string>
     <string name="MULTIBLOB">مُتعددة النُسَخ</string>
     <string name="Play_Nebulous_For_1_Year_">ألعب نيبلوس لمُدَّة 1 سنة!</string>
     <string name="Play_Nebulous_For_X_Years_">العب نيبلوس لمُدَّة %d سنة!</string>
-    <string name="Starting___">بدء…</string>
+    <string name="Starting___">بدأ…</string>
```

Comment:
The edit he made is wrong, the old translation is correct

Reply:
Can you explain to me more where the error is? 
If you are correct, I will delete this translation 
If you're taking this out of revenge, 
**stop bothering us**

Reply:
I fixed it